docs/components/connectors/out-of-the-box-connectors/amazon-textract.md

```diff
@@ -119,6 +119,10 @@ The response from the **Amazon Textract Connector** mirrors the AWS Textract ser
 - **[Polling Execution Response](https://docs.aws.amazon.com/textract/latest/dg/API_GetDocumentAnalysis.html#API_GetDocumentAnalysis_ResponseSyntax)**: Returns chunks of data in a paginated format for multi-page or complex documents.
 - **[Asynchronous Execution Response](https://docs.aws.amazon.com/textract/latest/dg/API_StartDocumentAnalysis.html#API_StartDocumentAnalysis_ResponseSyntax)**: Used for batch processing where results are returned later through job completion.
 
+:::note
+Starting from version 8.7.0, the Amazon Textract Connector supports storing the response as a document. See additional details and limitations in [document handling](/components/concepts/document-handling.md).
+:::
+
```

Comment:
For Textract: "... the Amazon Textract Connector can read the input document directly from the Camunda document store"